nuxeo/values.yaml

```diff
@@ -212,3 +212,17 @@ extraVolumes: []
 podSecurityContext:
   fsGroup: 1000
 containerSecurityContext: {}
+
+## PodDisruptionBudget configuration.
+podDisruptionBudget: {}
+  ## The minimum number of pods that must be available during a disruption.
+  ## Can be an absolute number (integer) or a percentage (string).
+  ## If set, `maxUnavailable` must not be present.
+  ## Example values: 1, 50%.
+  # minAvailable: "50%"
+
+  ## The maximum number of pods that can be unavailable during a disruption.
+  ## Can be an absolute number (integer) or a percentage (string).
+  ## If set, `minAvailable` must not be present.
+  ## Example values: 1, 50%.
+  # maxUnavailable: 1
```

Comment:
The documentation is good, could you update the two lines below to the exact key to configure in a `values.yaml`file?
```yaml
  # minAvailable: "50%"
  # maxUnavailable: 1
```

Reply:
Thank you. Updated